nixos/modules/services/hardware/undervolt.nix

```diff
@@ -32,58 +46,58 @@ in {
     };
 
     coreOffset = mkOption {
-      type = types.nullOr types.str;
+      type = types.nullOr types.int;
```

Comment:
Originally I was worried that moving to `int` would prevent us from using fractional values (i.e. "-75.5").

But looking at the undervolt source code these flags are parsed as integers anyway so this seems like a good change :+1: